### PR TITLE
feat(createCircuitBreaker): log circuit breaker activity

### PR DESCRIPTION
**Pull request by 10xLaCroixDrinker**
## Description

Log circuit breaker opens, closes and healthcheck failures

## Motivation and Context

Provides better insight and monitoring

## How Has This Been Tested?

Included unit tests

## Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation (adding or updating documentation)
- [ ] Dependency update

## Checklist:

- [ ] My change requires a change to the documentation and I have updated the documentation accordingly.
- [ ] These changes should be applied to a maintenance branch.
- [ ] This change requires cross browser checks.
- [ ] Performance tests should be ran against the server prior to merging.
- [ ] This change impacts caching for client browsers.
- [ ] This change impacts HTTP headers.
- [ ] This change adds additional environment variable requirements for One App users.
- [ ] I have added the Apache 2.0 license header to any new files created.

## What is the Impact to Developers Using One App?

Developers will see circuit breaker activities in their logs

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -161,6 +161,7 @@
     "nodemon": "^2.0.4",
     "ora": "^4.0.4",
     "parse-prometheus-text-format": "^1.1.1",
+    "prettier": "^2.0.5",
```
Comment (JAdshead) [inline package.json:164]: do we want to add prettier to one-app ?

Comment (10xLaCroixDrinker) [reply package.json:164]: This is a peerDep required for inline snapshots, thats all I added it for

**Comment by mtomcal** [comment]
This is great! But can we also send metrics as well?

**Comment by 10xLaCroixDrinker** [comment]
> This is great! But can we also send metrics as well?

We already do 🙂

**Pull request merged**